examples/tuning/cta_svm/main.py

```diff
@@ -100,3 +106,21 @@ def evaluate_pipeline():
 $ python main.py --tune_mode (pipeline/params) --species mouse --tissue Kidney --train_dataset 4682 --test_dataset 203
 
 """
+"""
+The 10 processes with the highest test_acc in step 2
```

Comment:
If you were to put tables like this in the source code, please make it readable. For example, show less decimal points (for the sake of an example, 3 decimal points should be enough) and align the columns.

Reply:
changed